README.md

```diff
@@ -43,6 +43,7 @@ Currently supported grammars are:
   * Python
   * RSpec
   * Racket
+  * [RANT](https://github.com/TheBerkin/Rant)
```

Comment:
None of these other languages have a link, so I'm not sure I want to introduce the link unless more are...